backend/src/test/java/de/neuefische/backend/service/UserServiceTest.java

```diff
@@ -50,17 +55,38 @@ void deleteUser_expect_correct_id() {
 
     @Test
     void findUserById_expect_throws_exception() {
-        assertThrows(NoSuchElementException.class,()->userService.findUserById("0"));
+        assertThrows(NoSuchElementException.class, () -> userService.findUserById("0"));
     }
 
     @Test
     void findUser_expect_correct_user() {
         //given
-        User givenUser = new User("0","hanna",new ArrayList<>());
+        User givenUser = new User("0", "hanna", new HashSet<>());
         //when
         when(userRepo.findById(givenUser.id())).thenReturn(Optional.of(givenUser));
         User userResult = userService.findUserById(givenUser.id());
         //then
-        assertEquals(givenUser,userResult);
+        assertEquals(givenUser, userResult);
+    }
+
+    @Test
+    void updateUser_expect_exception() {
```

Comment:
These names or not very meaning full, maybe you could add why you are expecting an exception and why not